modules/overrides/userconfigurableapi/client.go

```diff
@@ -118,12 +138,9 @@ func (o *userConfigOverridesClient) Get(ctx context.Context, userID string) (ten
 		}
 	}()
 
-	reader, _, err := o.r.Read(ctx, OverridesFileName, []string{OverridesKeyPath, userID}, false)
-	if err == backend.ErrDoesNotExist {
-		return nil, nil
-	}
+	reader, version, err := o.rw.ReadVersioned(ctx, OverridesFileName, []string{OverridesKeyPath, userID})
 	if err != nil {
-		return
+		return nil, "", err
 	}
```

Comment:
This is a change in behaviour: instead of returning `nil` on not found just pass through the `backend.ErrDoesNotExist` error. I thought this would be easier and make the client more of a simple passthrough component.